### DISCUSSION TITLE
SIP profile Gateway problem

### DISCUSSION THREAD

**Discussion by westice007** [hnimminh/libresbc#129] [Q&A] [answered]
Hello, I'm having some problems with my libresbc configuration, According to the configuration, https://github.com/jj1682000/librec_sbc_config_public/blob/master/minimal_access/sipprofile_FS_internal.json, The analog extranet can call the analog PBX, but the analog extranet terminal has no sound. After checking the sip command, it is found that When sbc outbound sends an ACK to the pbx, it is sent to the WAN ip (218.77.104.71), which I didn't set anywhere, it should be sent to the gateway ip (172.18.10.196)

The sbc analog internal ip address is 172.18.10.21 (outbound). The analog external ip address is 192.168.137.96 (inbound) and the pbx(gateway) 172.18.10.196
![image](https://github.com/hnimminh/libresbc/assets/6076502/1ad5f620-e010-4b73-9b02-536dad81c9eb)

**Comment by hnimminh** [comment, marked as answer]
Likely IP address in **contact** header in 200 OK from callee's device (172.18.10.196) is 218.77.104.71
This should be fixed by config your gateway
